config-linux.md

```diff
@@ -270,13 +270,16 @@ For more information, see the kernel cgroups documentation about [memory][cgroup
 **`memory`** (object, OPTIONAL) represents the cgroup subsystem `memory` and it's used to set limits on the container's memory usage.
 For more information, see the kernel cgroups documentation about [memory][cgroup-v1-memory].
 
+Values specify the limit in bytes, or `-1` for unlimited memory. For `swappiness` the values are from `0` (almost never swap) to `100`
+(very likely to be swapped).
```

Comment:
Your newline is in the middle of the second sentence, but should be after the first period to match [our policy][1].

I'd rather have the `swappiness` semantics documented in the `swappiness` entry below (or dropped entirely if you feel the current “sysctl's vm.swappiness” punt is sufficient).  Then you can use:

> For all parameters except `swappiness`, values specify…

And in the entries themselves, drop the now-redundant “bytes”.  For example:

> **`limit`** *(int64, OPTIONAL)* - limit memory usage
> **`reservation`** *(int64, OPTIONAL)* - soft limit of memory usage
> …

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc5/style.md#one-sentence-per-line